src/pprof.c

```diff
@@ -520,8 +520,6 @@ DProf *pprof_Init(DProf *dp, const char **sample_names,
     if (!dp)
       return NULL;
     dp->ownership = 1;
-  } else {
-    dp->ownership = 0;
```

Comment:
This one also went away; intentional?

Reply:
It was intentional, but wrong.  Should be fixed, but there may be other instances.  I'll double-check.